services/workflows-service/prisma/schema.prisma

```diff
@@ -333,6 +334,29 @@ model SalesforceIntegration {
   idUrl               String   @db.VarChar(500)
 }
 
+enum UiDefinitionContext {
```

Comment:
The usage of the word `context` considering existing connotation to a workflow's context is something we should consider.